### ISSUE TITLE
Excluded Counties

### ISSUE THREAD

**Issue by demonstrous** [nytimes/covid-19-data#7]
You may not be aware but the boroughs of New York City are also counties. Brooklyn=Kings Queens=Queens, Staten Island= Richmond, Bronx=Bronx, Manhattan=New York (obviously problematic naming).  The population of Kings  county is larger than at least 4 western states.  Please consider adding them to your account.  The daily data can be found here: https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-daily-data-summary.pdf.

I live in Brooklyn and I have personally been tracking NY state county numbers here: https://docs.google.com/spreadsheets/d/159YnHmWplkEkhUZbC5oFT9cKiibLVMzaRm4LbhHN4Co/edit?usp=sharing

Thanks for your efforts,
Cooper Miller

**Comment by jicruz96** [comment]
Territories such as Puerto Rico don't subdivide into counties, but their municipal districts serve just as as well. Why not use these as proxies? This will allow better geographical specificity in the data for our fellow Americans in the territories.

**Comment by jessieb2001** [comment]
Vernon County, Missouri is missing from the data.

**Comment by jqnatividad** [comment]
+100 on breaking down NYC count into the 5 boroughs/counties, especially since NYC accounts for  the bulk of covid-19 cases in the US, never mind NYT is the hometown paper.

**Comment by cgsantucci** [comment]
> You may not be aware but the boroughs of New York City are also counties. Brooklyn=Kings Queens=Queens, Staten Island= Richmond, Bronx=Bronx, Manhattan=New York (obviously problematic naming). The population of Kings county is larger than at least 4 western states. Please consider adding them to your account. The daily data can be found here: https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-daily-data-summary.pdf.
> 
> I live in Brooklyn and I have personally been tracking NY state county numbers here: ://docs.google.com/spreadsheets/d/159YnHmWplkEkhUZbC5oFT9cKiibLVMzaRm4LbhHN4Co/edit?usp=sharing
> 
> Thanks for your efforts,
> Cooper Miller

Hello Cooper,
I have collected data by borough back to March 19 on cases and am looking for any previous boro disaggregated data to create density progression maps using GIS. By any chance do you have these numbers? Happy to share info I've collected if that is helpful. 
Best, Christina

**Comment by dannguyen** [comment]
I agree that splitting NYC into its separate counties would make data analysis/viz slightly easier, but I assume they grouped the 5 counties together because that matches the actual reality of the NYC bureaucracy. AFAIK, the 5 counties do not have their own governments, and so there isn't as much real-world importance to treat Kings and Manhattan County as separate entities, in the way it does for, say, Nassau and Suffolk counties, which each have their own government and bureaucracy accountable for multiple cities

**Comment by demonstrous** [comment]
> > You may not be aware but the boroughs of New York City are also counties. Brooklyn=Kings Queens=Queens, Staten Island= Richmond, Bronx=Bronx, Manhattan=New York (obviously problematic naming). The population of Kings county is larger than at least 4 western states. Please consider adding them to your account. The daily data can be found here: https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-daily-data-summary.pdf.
> > I live in Brooklyn and I have personally been tracking NY state county numbers here: ://docs.google.com/spreadsheets/d/159YnHmWplkEkhUZbC5oFT9cKiibLVMzaRm4LbhHN4Co/edit?usp=sharing
> > Thanks for your efforts,
> > Cooper Miller
> 
> Hello Cooper,
> I have collected data by borough back to March 19 on cases and am looking for any previous boro disaggregated data to create density progression maps using GIS. By any chance do you have these numbers? Happy to share info I've collected if that is helpful.
> Best, Christina

I started to look at NYS counties on March 10th.  I started looking into city data on the 19th, at some point the city gov site got a facelift and the data was missing for a day.  I went to the wayback machine on archive.org to look at previous versions of the information, 18th was as far back as they were sharing data online.
March 18th:
Unknown | 1
Staten Island | 42
Bronx | 106
Brooklyn | 236
Queens | 248
Manhattan | 290
Total | 923

More city info is here:
https://www1.nyc.gov/site/doh/covid/covid-19-providers.page

I just found this neighborhood heat map(!) on that page:
https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-data-map.pdf

It might be worth checking the wayback machine to see if the pdf was updated.
Let me know if this helps, Christina!
-Cooper

**Comment by demonstrous** [comment]
(didn't mean to close this thread)

**Comment by beighleyashley** [comment]
I believe, through the process of elimination, the “Unknown” county in California is Tuolumne. We had our first confirmed case on the 25th (patient was actually from Mono county, but had traveled to Tuolumne county the day prior to being admitted into the hospital, and supposedly had little/no contact with other residents, except one companion who is not showing signs, but being quarantined.) I may be wrong, but either way, Tuolumne is missing from the list.

**Comment by cgsantucci** [comment]
> > > You may not be aware but the boroughs of New York City are also counties. Brooklyn=Kings Queens=Queens, Staten Island= Richmond, Bronx=Bronx, Manhattan=New York (obviously problematic naming). The population of Kings county is larger than at least 4 western states. Please consider adding them to your account. The daily data can be found here: https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-daily-data-summary.pdf.
> > > I live in Brooklyn and I have personally been tracking NY state county numbers here: ://docs.google.com/spreadsheets/d/159YnHmWplkEkhUZbC5oFT9cKiibLVMzaRm4LbhHN4Co/edit?usp=sharing
> > > Thanks for your efforts,
> > > Cooper Miller
> > 
> > 
> > Hello Cooper,
> > I have collected data by borough back to March 19 on cases and am looking for any previous boro disaggregated data to create density progression maps using GIS. By any chance do you have these numbers? Happy to share info I've collected if that is helpful.
> > Best, Christina
> 
> I started to look at NYS counties on March 10th. I started looking into city data on the 19th, at some point the city gov site got a facelift and the data was missing for a day. I went to the wayback machine on archive.org to look at previous versions of the information, 18th was as far back as they were sharing data online.
> March 18th:
> Unknown | 1
> Staten Island | 42
> Bronx | 106
> Brooklyn | 236
> Queens | 248
> Manhattan | 290
> Total | 923
> 
> More city info is here:
> https://www1.nyc.gov/site/doh/covid/covid-19-providers.page
> 
> I just found this neighborhood heat map(!) on that page:
> https://www1.nyc.gov/assets/doh/downloads/pdf/imm/covid-19-data-map.pdf
> 
> It might be worth checking the wayback machine to see if the pdf was updated.
> Let me know if this helps, Christina!
> -Cooper

Belated thank you for the 18th data!

**Comment by cgsantucci** [comment]
Belated thank you for the 18th numbers!

**Comment by albertsun** [comment]
@demonstrous We are aware that the boroughs are counties. New York State reports data for New York City as a single entity. The city has been reporting data by borough separately, but we are still checking to see that the way it reports cases is consistent with how cases are reported across the rest of the dataset before integrating borough-level numbers into our dataset.

@jessieb2001 Vernon County, Missouri is not in the data because it has 0 cases and 0 deaths. All counties without any cases or deaths are excluded.

@jicruz96 Thanks for that suggestion. We haven't seen Puerto Rico report any data at the municipal district level yet though.

**Comment by m0L3hMu8Y** [comment]
Hi there. This is incredibly helpful, and a great service for all of us. If you find it valid, I believe there are 3 cases in Jasper County, Texas, which hasn’t yet made the list. Here’s the link to a local news source. https://www.kjas.com/news/local_news/article_e3e2af3a-7458-11ea-945c-8bdc1a134ef5.html

Thanks again!

**Comment by albertsun** [comment]
Hi @m0L3hMu8Y, thanks for passing that source along. For the time being we are going to be counting Jasper County, Texas with two cases as that is the number the state health department is still reporting. That will be in the next release of data for 4/2.
